src/openrct2-ui/ride/Construction.cpp

```diff
@@ -24,6 +24,209 @@ using namespace OpenRCT2::TrackMetaData;
 
 namespace OpenRCT2
 {
+    constexpr auto kSeparator = TrackElemType::None;
+
+    /**
+     * Order of special track elements dropdown. Elements with the same name string must be sequential or they show up twice.
+     */
+    constexpr std::array kSpecialElementsDropdownOrder = {
+        TrackElemType::EndStation,
+        kSeparator,
+
+        // Brakes
+        TrackElemType::Brakes,
+        TrackElemType::DiagBrakes,
+        TrackElemType::Down25Brakes,
+        TrackElemType::BlockBrakes,
+        TrackElemType::DiagBlockBrakes,
+        kSeparator,
```

Comment:
The separators seem like a good idea, and the implementation is very clean. I'm not sure about their value ingame. On the B&M rides, there are 13 separators, which inflates the number of items by 37%. Car ride has 2 separators for 3 elements, which is a 67% increase.

I think the purpose of separators - to group the elements by function - the new order does pretty well already.

![image](https://github.com/user-attachments/assets/f6ca121a-f703-4b96-a84c-e8c2ed5560ae)

![image](https://github.com/user-attachments/assets/93eefd27-fdf6-427d-ad40-e5eec44db9fa)

Reply:
After playing around with the separators for a bit, I think two is the ideal number: one to divide BrakeForDrop from S-Bends and one to divide Thrust Air pieces from inversions.